src/backend/distributed/commands/type.c

```diff
@@ -489,7 +523,16 @@ AlterTypeOwnerObjectAddress(Node *node, bool missing_ok)
 	Assert(stmt->objectType == OBJECT_TYPE);
 
 	TypeName *typeName = makeTypeNameFromNameList((List *) stmt->object);
-	Oid typeOid = LookupTypeNameOid(NULL, typeName, missing_ok);
+	Oid typeOid = LookupTypeNameOid(NULL, typeName, true);
+	if (!OidIsValid(typeOid))
+	{
+		/*
+		 * Citus should not throw error for non-existing objects, let Postgres do that.
+		 * Otherwise, Citus might throw a different error than Postgres, which we don't want.
+		 */
+		return InvalidObjectAddress;
```

Comment:
there are bunch of test cases that are NOT covered by code-cov. Can you please make sure to cover those?


(Note that maybe we have NOT enabled code-cov for vanilla testing. If not, it'd be great to enable -- though not sure how we enable it).